### ISSUE TITLE
Version number link

### ISSUE THREAD

**Issue by yiwenxx** [pulibrary/aux#32]
### **Issue**

1. Version number link should be white in color instead of blue, failing color contrast accessibility.
2. No spacing between version information and divider line.

<img width="1511" alt="Image" src="https://github.com/user-attachments/assets/56fd3098-d417-4843-a80b-9de3f7f0986b" />

### **Implementation Notes**

1. Use the same white that the text is using
2. Nice to have: add a 8px spacing in between version information and divider.
<img width="916" alt="Image" src="https://github.com/user-attachments/assets/87b5a5cd-c6ab-403c-88ea-051e19e24d65" />

**Comment by matthewjchandler** [comment]
Looks ready to send to RDSS. Also high priority because public-facing
